camp/ocampbell/xBoxTriggerProject/src/main/java/frc/robot/commands/ControlledFalconDefault.java

```diff
@@ -0,0 +1,88 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// File: ControlledFalconDefault.java
+// Intent: Assigns a motor speed value to a falcon.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+package frc.robot.commands;
+
+import frc.robot.control.SubsystemCollection;
+import frc.robot.subsystems.ControlledFalcon;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ControlledFalconDefault extends CommandBase {
+
+  private SubsystemCollection subsystems;
+
+
+  private ControlledFalcon motor;
+  private double speed = 0.0;
+  private boolean done = false;
+  private double lTrigger = 0.0;
+  private double rTrigger = 0.0;
+
+
+  public ControlledFalconDefault(SubsystemCollection collection, ControlledFalcon motor, double lTrigger, double rTrigger) {
+      this.motor = motor;
+      this.lTrigger = lTrigger;
+      this.rTrigger = rTrigger;
+
+    addRequirements(motor);
+    subsystems = collection;
+}
+
+  @Override
+  public void initialize() {}
+
+
+  @Override
+  public void execute() {
+
+    lTrigger = -lTrigger;
+
+    //delta approach
+    speed = lTrigger + rTrigger;
+    speed = Math.max(-1.0, Math.min(1.0, speed));
+
+    //left trigger wins 
+    if(Math.abs(lTrigger) > rTrigger){
+      speed = lTrigger;
+    } else {
+      speed = rTrigger;
+    }
+
+    //right trigger wins 
+    if(rTrigger >= Math.abs(lTrigger)){
+      speed = rTrigger;
+    } else {
+      speed = lTrigger;
+    } 
+
+    //gta approach (favors right trigger but if both are pulled it breaks(speed = 0))
+    if(rTrigger > 0){
+      speed = rTrigger;
+    } else if (rTrigger == Math.abs(lTrigger)){   //possibly needs deadband
+      speed = 0;
+    } else if (Math.abs(lTrigger) > 0) {
+      speed = lTrigger;
+    } 
```

Comment:
So lots of approaches above ... can you tell me which one will actually win?

Reply:
Overall idea is to settle on one approach for calculating speed based on a set of requirements/principles.

As an FYI - many times in software development you as the author get to decide how things will work when the requirements are 'unwritten' (and trust me so many times requirements don't exist when they are actually necessary).  Meaning, the buck stops with you.  You need to decide what is the right behavior and be ready to back up why it is that you made that decision.  Personally, I like to put in comments why I'm making a decision.  Sometimes its as simple as: // TODO - for now assume that X is Y so we can ship anything, need to have meeting on X and Y ...   Other times I have clear opinions and will list reasons behind the chosen approach.